cleanrl/sac_continuous_action.py

```diff
@@ -276,7 +276,7 @@ def get_action(self, x):
                     if args.autotune:
                         with torch.no_grad():
                             _, log_pi, _ = actor.get_action(data.observations)
-                        alpha_loss = (-log_alpha * (log_pi + target_entropy)).mean()
+                        alpha_loss = (-log_alpha.exp() * (log_pi + target_entropy)).mean()
```

Comment:
I have taken the liberty of changing `torch.exp(log_alpha)` to log_alpha.exp() which is slight _cleaner_ :sweat_smile: 
I recall doing it that way before, but it was likely lost during refactors.

Reply:
Works for me.